apps/fluent-tester/package-lock.json

```diff
@@ -0,0 +1,2013 @@
+{
```

Comment:
We shouldn't need this file? Given we use `yarn` and not nom?

Reply:
Removing this - not sure why it was added :)